pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -121,7 +121,7 @@ func (c *Controller) rejectRequest(pktIn *ofctrl.PacketIn) error {
 		if c.antreaProxyEnabled {
 			matches := pktIn.GetMatches()
 			if match := getMatchRegField(matches, openflow.ServiceEPStateField); match != nil {
-				if svcEpstate, err := getInfoInReg(match, openflow.ServiceEPStateField.GetRange().ToNXRange()); err != nil {
+				if svcEpstate, err := getInfoInReg(match, openflow.ServiceEPStateField.GetRange().ToNXRange()); err == nil {
 					return svcEpstate&openflow.EpSelectedRegMark.GetValue() == openflow.EpSelectedRegMark.GetValue()
 				}
```

Comment:
I think the style we use in Antrea is more like this:
```go
if svcEpstate, err := getInfoInReg(match, openflow.ServiceEPStateField.GetRange().ToNXRange()); err != nil {
	return false
}
return svcEpstate&openflow.EpSelectedRegMark.GetValue() == openflow.EpSelectedRegMark.GetValue()
```
i.e. we try to handle the error case first

Reply:
Done.